src/supermarket/app/helpers/supported_platforms_helper.rb

```diff
@@ -1,4 +1,39 @@
 module SupportedPlatformsHelper
+
+  #
+  # Returns an array of all known platforms to Supermarket
+  #
+  # @example
+  #   puts known_platforms.join(',')
+  #
+  #
+  # @return [Array] platforms
+  #
+  def known_platforms
+    %(aix
+      amazon
+      cento
+      debian
+      fedora
+      freebsd
+      gentoo
+      mac_os_x
+      omnios
+      openbsd
+      opensuse
+      opensuseleap
+      oracle
+      redhat
+      ubuntu
+      scientific
+      smartos
+      solaris
+      suse
+      windows
+      zlinux)
+  end
```

Comment:
I have a not-fully-formed feeling that this canonical list of known platform names should live deeper, like as a class constant in the `SupportedPlatforms` model. That could still be bubbled up to views through this helper method.